libs/partners/cohere/langchain_cohere/__init__.py

```diff
@@ -0,0 +1,12 @@
+from langchain_cohere.chat_models import ChatCohere
+from langchain_cohere.embeddings import CohereEmbeddings
+from langchain_cohere.llms import BaseCohere
+from langchain_cohere.rag_retrievers import CohereRagRetriever
+
+__all__ = [
+    "BaseCohere",
```

Comment:
probably don't need/want to export BaseCohere here? not something for end users i assume